services/metadata_service/api/artifact.py

```diff
@@ -59,6 +60,7 @@ def __init__(self, app):
             self.create_artifacts,
         )
         self._async_table = AsyncPostgresDB.get_instance().artifact_table_postgres
+        self._async_run_table = AsyncPostgresDB.get_instance().run_table_postgres
```

Comment:
This will be a common pattern.  Various API modules now require the run table.  Not a new pattern.  `TaskApi` already did this.